packages/eslint-plugin-lit-a11y/lib/rules/accessible-emoji.js

```diff
@@ -5,8 +5,8 @@
 
 const emojiRegex = require('emoji-regex');
 const ruleExtender = require('eslint-rule-extender');
-const { TemplateAnalyzer } = require('../../template-analyzer/template-analyzer.js');
```

Comment:
CI format please

Reply:
this one triggered it
we should add prettier action problem matcher thingy to get highlights on lines with errors

Reply:
https://github.com/open-wc/open-wc/actions/runs/1311057217 it says "action required"

not exactly sure which action it requires though :D

Reply:
https://github.com/open-wc/open-wc/runs/3812670994?check_suite_focus=true

🤷‍♂️ I blame James